src/assets/themes/ngenius-theme.scss

```diff
@@ -0,0 +1,235 @@
+/**
```

Comment:
we'll probably want to revisit this and pull all the `#` colors out into scss variables, but no rush for now.